src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -2,104 +2,63 @@
 out: Basic-Def.html
 ---
 
-  [More-About-Settings]: More-About-Settings.html
+  [Task-Graph]: Task-Graph.html
   [Bare-Def]: Bare-Def.html
   [Full-Def]: Full-Def.html
   [Running]: Running.html
   [Library-Dependencies]: Library-Dependencies.html
   [Input-Tasks]: ../docs/Input-Tasks.html
 
-.sbt build definition
----------------------
+Build definition
+----------------
 
 This page describes sbt build definitions, including some "theory" and
 the syntax of `build.sbt`. It assumes you know how to [use sbt][Running]
 and have read the previous pages in the Getting Started Guide.
 
-### Three Flavors of Build Definition
-
-There are three flavors of build definition:
-
-1. Multi-project `.sbt` build definition
-2. Bare `.sbt` build definition
-3. `.scala` build definition
-
-This page discusses the newest multi-project `.sbt` build definition, which combines the strength
-of the two older flavors, and is suitable for all cases.
-You might come across the other older flavors when dealing with builds in the wild.
-See [bare .sbt build definition][Bare-Def] and [.scala build definition][Full-Def] (later in Getting Started) for more
-on other flavors.
-
-In addition, a build definition can contain files ending in `.scala`, located in the
-`project/` subdirectory of the base directory to define commonly used functions and values.
+This page discusses the `build.sbt` build definition.
 
 ### What is a Build Definition?
 
 After examining a set of directories and processing build definition files, sbt
-ends up with `Project` definitions.
+ends up with a set of `Project` definitions.
```

Comment:
I think this is slightly confusing because earlier, `project` was used to refer to the whole thing that was created by `sbt new`, and now we have multiple `Project`s inside the original `project`. Could we pick different words to distinguish these two concepts? Maybe call the inner ones sub-`Project`s or something? Consistently call the outer one a `build`? I don't know what would be best

This certainly confused me when I first started using SBT.